src/scaffoldmaker/meshtypes/meshtype_3d_lung1.py

```diff
@@ -780,3 +684,496 @@ def refineMesh(cls, meshrefinement, options):
+
+                if (e1 == 1) and (e3 == (elementsCount3 - 1)):
+                    nodeIdentifiers.pop(6)
+                    nodeIdentifiers.pop(4)
+                    eft = eftfactory.createEftBasic()
+                    setEftScaleFactorIds(eft, [1], [])
+                    nodes = [5, 6, 7, 8]
+                    collapseNodes = [5, 7]
+                    remapEftNodeValueLabel(eft, nodes, Node.VALUE_LABEL_D_DS1, [])
+                    remapEftNodeValueLabel(eft, collapseNodes, Node.VALUE_LABEL_D_DS3,
+                                           [(Node.VALUE_LABEL_D_DS1, []), (Node.VALUE_LABEL_D_DS3, [])])
+                    if e2 == 0:
+                        # Remapping the element
+                        remapEftNodeValueLabel(eft, [3], Node.VALUE_LABEL_D_DS2,
+                                               [(Node.VALUE_LABEL_D_DS2, []), (Node.VALUE_LABEL_D_DS1, [1])])
+                    ln_map = [1, 2, 3, 4, 5, 5, 6, 6]
+                    remapEftLocalNodes(eft, 6, ln_map)
+
+                elif (e1 == 1) and (e2 == 0) and (e3 < (elementsCount3 - 1)):
+                    # Remapping the elements
+                    eft = eftfactory.createEftBasic()
+                    setEftScaleFactorIds(eft, [1], [])
+                    remapEftNodeValueLabel(eft, [3, 7], Node.VALUE_LABEL_D_DS2,
+                                           [(Node.VALUE_LABEL_D_DS2, []), (Node.VALUE_LABEL_D_DS1, [1])])
+
+                elif (e1 == elementsCount1 - 1) and (e3 == (elementsCount3 - 1)):
+                    nodeIdentifiers.pop(7)
+                    nodeIdentifiers.pop(5)
+                    eft = eftfactory.createEftBasic()
+                    setEftScaleFactorIds(eft, [1], [])
+                    nodes = [5, 6, 7, 8]
+                    collapseNodes = [6, 8]
+                    remapEftNodeValueLabel(eft, collapseNodes, Node.VALUE_LABEL_D_DS3,
+                                           [(Node.VALUE_LABEL_D_DS1, [1]), (Node.VALUE_LABEL_D_DS3, [])])
+                    remapEftNodeValueLabel(eft, nodes, Node.VALUE_LABEL_D_DS1, [])
+                    ln_map = [1, 2, 3, 4, 5, 5, 6, 6]
+                    remapEftLocalNodes(eft, 6, ln_map)
+                    # Remapping the elements
+                    if e2 == 0:
+                        remapEftNodeValueLabel(eft, [3], Node.VALUE_LABEL_D_DS2,
+                                               [(Node.VALUE_LABEL_D_DS2, []), (Node.VALUE_LABEL_D_DS1, [1])])
+                        remapEftNodeValueLabel(eft, [3], Node.VALUE_LABEL_D_DS1, [(Node.VALUE_LABEL_D_DS2, [])])
+                    elif e2 == 1:
+                        remapEftNodeValueLabel(eft, [1], Node.VALUE_LABEL_D_DS2,
+                                               [(Node.VALUE_LABEL_D_DS2, []), (Node.VALUE_LABEL_D_DS1, [1])])
+                        remapEftNodeValueLabel(eft, [1], Node.VALUE_LABEL_D_DS1, [(Node.VALUE_LABEL_D_DS2, [])])
```

Comment:
Are these lines needed? May be the cause of the inconsistency between elements I showed in slack, when you view the refined mesh with exterior lines.